README.md

```diff
@@ -82,7 +82,7 @@ Run with coverage, missing lines and junit output `python -m pytest --cov-report
 
 ## Yara rules checks.
 
-There is a Jenkins job [TDR Antivirus Rule Update Check](https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Rule%20Update%20Check/) which is run on a schedule from the Jenkinsfile-check-rules file. This carries out the following steps. 
+There is a GitHub actions job [TDR Check Antivirus Rules](https://github.com/nationalarchives/tdr-antivirus/actions/workflows/check_rules.yml) which is run on a schedule from the .github/workflows/check_rules file. This carries out the following steps. 
```

Comment:
Maybe mention the time the schedule is run?

Reply:
Yep makes sense, I've added it in.